tests/virtualization/virtman_storage.pm

```diff
@@ -75,7 +76,7 @@ sub create_nfs_share {
     send_key "ret";
     type_string "echo '$dir *(rw,sync)' >> /etc/exports";
     send_key "ret";
-    type_string "systemctl restart nfsserver.service";
+    systemctl 'restart nfsserver';
```

Comment:
@nicksinger is more concrete ... `systemctl restart something` also find `something.timer|target|socket`